### DISCUSSION TITLE
Not all available Adwaita icons are applied

### DISCUSSION THREAD

**Discussion by isf63** [lxqt/lxqt#2505] [Q&A] [answered]
Upon creating a few PRs to add menus icons coexisting in the Adwaita and KDE icon themes to LXQt, I noticed when applying the Adwaita theme from Appearance Config that some icons do not get applied despite existing in the theme.

Examples:
- The top-right New Tab button in PCManFM-Qt is not applied - exists in `.../Adwaita/symbolic/actions/tab-new-symbolic.svg`.

- Menu icons are hardly populated with Adwaita - icons such as `edit-cut`, `edit-copy`, and `edit-paste` are not applied. However, they exist in `.../Adwaita/symbolic/actions/*-symbolic.svg`.

- Many more...

I don't know if this is reproducible or just on my installation. However, it may affect other icon themes besides Adwaita as well.

##### Possible Solution
Apply symbolic variants when non-symbolic icons are not available.

##### Steps to Reproduce (for bugs)
1. LXQt Appearance Config -> Apply Adwaita icons theme.
2. See examples above.

##### System Information
* Distribution & Version: Arch Linux
* Qt Version: 5.15.11+kde+r147
* liblxqt Version: 1.4.0

**Comment by tsujan** [comment, marked as answer]
LXQt follows `freedesktop.org`, not GNOME. In no way `tab-new-symbolic` should be used instead of `tab-new`.

**Comment by tsujan** [comment]
I converted this to a discussion/question because it might be helpful to others.

Also, the following explanation may remove misunderstandings:

In LXQt (like in KDE), really symbolic icons — whose colors change based on the colors of the active Qt style — are SVG icons distinguished by their internal stylesheets, not by their names. It's the icon theme that determines where a symbolic icon is used (usually where its size is small).

Of course, GNOME can use any method to have small, color-changing icons with GTK, but LXQt does't break `freedesktop.org` rules with Qt.

Installing an icon set that supports SVG symbolic icons is recommended (although not necessary). There are many besides Breeze. GNOME's "-symbolic" icons aren't symbolic outside GNOME/GTK.
